src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaver.shared.cs

```diff
@@ -29,6 +29,24 @@ public static class FileSaver
 	[SupportedOSPlatform("Windows")]
 	public static Task<FileSaverResult> SaveAsync(string fileName, Stream stream, CancellationToken cancellationToken = default) =>
 		Default.SaveAsync(fileName, stream, cancellationToken);
+	
+	/// <inheritdoc cref="IFileSaver.SaveAsync(string, string, Stream, IProgress{double}, CancellationToken)"/>
+	[SupportedOSPlatform("Android26.0")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
+	[SupportedOSPlatform("Tizen")]
+	[SupportedOSPlatform("Windows")]
+	public static Task<FileSaverResult> SaveAsync(string initialPath, string fileName, Stream stream, IProgress<double> progress, CancellationToken cancellationToken = default) =>
+		Default.SaveAsync(initialPath, fileName, stream, cancellationToken);
+
+	/// <inheritdoc cref="IFileSaver.SaveAsync(string, Stream, IProgress{double}, CancellationToken)"/>
+	[SupportedOSPlatform("Android")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
+	[SupportedOSPlatform("Tizen")]
+	[SupportedOSPlatform("Windows")]
+	public static Task<FileSaverResult> SaveAsync(string fileName, Stream stream, IProgress<double> progress, CancellationToken cancellationToken = default) =>
+		Default.SaveAsync(fileName, stream, cancellationToken);
```

Comment:
Looks like you missed to pass the `progress` parameter on `Default.SaveAsync` method